test/tests/morebits.array.js

```diff
@@ -0,0 +1,15 @@
+QUnit.module('Morebits.array');
+QUnit.test('chunk', assert => {
+	assert.deepEqual(Morebits.array.chunk([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 3), [[1, 2, 3], [4, 5, 6], [7, 8, 9], [10]], '10 over 3');
+	assert.deepEqual(Morebits.array.chunk([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 42), [[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]], 'Size > length');
+	assert.deepEqual(Morebits.array.chunk([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]), [[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]], 'No size');
+	// assert.throws(Morebits.array.chunk(42), 'throws');
```

Comment:
In case you were wondering, using `throws()` takes the statement as a code block. Right now it would execute outside the test and not possible for QUnit to check it. (See <https://api.qunitjs.com/assert/throws/>.)

Like so:

```js
assert.throws(function() {
  Morebits.array.chunk(42);
}, 'No array');
```

Or pick one of the shorter styles that JavaScript allows:

```js
assert.throws(
  () => { Morebits.array.chunk(42); },
  'No array'
);

assert.throws(() => Morebits.array.chunk(42), 'No array');
```

Reply:
Awesome, thanks!  Done!